tests/pytest.ini

```diff
@@ -3,6 +3,7 @@ addopts = -s -v
 markers =
     sanity: marks tests as sanity
     dut: mark tests with dut which need dut
+    skip_if_athena: marks the tests as athena not supported
```

Comment:
don't add `athena` specific skip marker. Add a test marked which indicates that the test uses `device` configuration.

If any test doesn't support devices, it can skip that by saying, `run all tests except device tests`

Reply:
changed to device